entrypoint.sh

```diff
@@ -1,10 +1,11 @@
 #!/bin/sh
 
-if [ "$accountname" != "demo" ] 
+if [ "$ACCOUNT" != "demo" ] 
 then
-	./collect_data.sh --account $accountname
+  pipenv run python cloudmapper.py gather --account-name $ACCOUNT
 else
 	cp config.json.demo config.json
 fi
-pipenv run python cloudmapper.py prepare --account $accountname
+pipenv run python cloudmapper.py prepare --account $ACCOUNT
```

Comment:
needs to be --account-name
you can see it here 
https://github.com/duo-labs/cloudmapper/pull/54/files

Reply:
Will fix- give me a minute